File: 3222-find-the-winning-player-in-coin-game/3222-find-the-winning-player-in-coin-game.java
class Solution {
    public String winningPlayer(int x, int y) {
        if(y<4) return "Bob";
        // int moves=x<(y/4)?x:(y/4);
        if((x<(y/4)?x:(y/4))%2 == 0){
            return "Bob";
        }
        return "Alice";

        
    }
}

File: 3222-find-the-winning-player-in-coin-game/3222-find-the-winning-player-in-coin-game.py
class Solution(object):
    def winningPlayer(self, x, y):
        if y<4: return "Bob"
        moves=0
        if x < (y//4):
            moves=x
        else:
            moves=y//4
        if moves%2 == 0:
            return "Bob"
        return "Alice"
        